### PR TITLE
Update classes that implement ArrayAccess functions for PHP 8

### PR DESCRIPTION
**Pull request by quangdo-aligent**
Update composer.json version and php requirements to reflect this.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "zip/magento2",
     "description": "Zip Payment Module for Magento 2",
-    "version": "1.1.11",
+    "version": "1.1.12",
```
Comment (aligent-lturner) [inline composer.json]: This should be a new major version, as it's not backwards-compatible

**Review by aligent-lturner — CHANGES_REQUESTED** [review]
@quangdo-aligent just one comment on the composer.json version

File: composer.json (hunk 1 above)
Comment (quangdo-aligent) [reply composer.json]: :+1: Good point.

Comment (quangdo-aligent) [reply composer.json]: Done

**Review by aligent-lturner — APPROVED** [review]

**Pull request merged**